_workinggroups/generators.md

```diff
@@ -3,26 +3,27 @@ title: "Physics Generators"
 layout: plain
 ---
 
-The Physics Generators Working Group is a common forum for discussion and 
+The Physics Generators Working Group is a common forum for discussion and
 technical work on the physics event generators used by HEP experiments.
-It promotes the collaboration of experimental and theoretical physicists from 
-different experiments and generator teams and of software and computing engineers,
-with the aim of having them work together on improving the current codes 
-and production workflows and on making new theoretical advances 
+It promotes the collaboration of experimental and theoretical physicists
+from different experiments and generator teams and of software and computing
+engineers, with the aim of having them work together on improving the current
+codes and production workflows and on making new theoretical advances
 easier to implement in a computationally efficient way.
 
-The working group was set up as a follow-up of the Physics Event Generator 
-Computing Workshop that was held at CERN in November 2018, during which 
+The working group was set up as a follow-up of the Physics Event Generator
+Computing Workshop that was held at CERN in November 2018, during which
 an initial program of work in this area was identified.
-The working group is currently coordinated by Andrea Valassi, Josh McFayden, 
-Stefan Hoeche, Steve Mrenna and Taylor Childers.
-All coordinators can be reached 
+
+The working group is currently coordinated by Andrea Valassi,
+Josh McFayden, Stefan Hoeche, Steve Mrenna and Taylor Childers.
+All coordinators can be reached
 via <hsf-generator-wg-convenors@googlegroups.com>.
 
-The mailing list for all WG members is hsf-generator-wg@googlegroups.com.
-If you would like to subscribe to this group, 
-please click on https://groups.google.com/group/hsf-generator-wg/subscribe 
-or send an email to hsf-generator-wg+subscribe@googlegroups.com.
+The mailing list for all WG members is <hsf-generator-wg@googlegroups.com>.
+If you would like to subscribe to this group,
+please click on <https://groups.google.com/group/hsf-generator-wg/subscribe>
+or send an email to <hsf-generator-wg+subscribe@googlegroups.com>.
```

Comment:
For me this last one is not being rendered as a link - maybe the `+` confuses the conversion from markdown in Jekyll?

Maybe try:

[send an email to subscribe](hsf-generator-wg+subscribe@googlegroups.com)

What do you get when you build the page locally?

Reply:
Hi Graeme, indeed I was about to submit a fix for that: "send an email to [hsf-generator-wg+subscribe@googlegroups.com](mailto:hsf-generator-wg+subscribe@googlegroups.com)", which should force this the hard way.

Sorry for all this mess :-(

I am not doing a local build, I usually rely on visualizing the md in github itself, and there all looks good: https://github.com/valassi/hep-sf.github.io/blob/789f0bef7b33f749207291719bfa3138faf04c51/_workinggroups/generators.md
The problem must be the dialect difference between github md and jekyll md..?

How can I do a local build, is this documented on the site?

By the way, is it ok if I self-approve these fixes? 

Cheers Andrea

Reply:
Created a MR

https://github.com/HSF/hsf.github.io/pull/490/files

Reply:
Ok marged and now it looks good now on the deployed site.

Reply:
> Sorry for all this mess :-(

Learning experience!

> 
> I am not doing a local build, I usually rely on visualizing the md in github itself, and there all looks good: https://github.com/valassi/hep-sf.github.io/blob/789f0bef7b33f749207291719bfa3138faf04c51/_workinggroups/generators.md
> The problem must be the dialect difference between github md and jekyll md..?

Yes, it’s a bit of an odd one, but these are murky corners in the world of Markdown. AFAIK Jekyll uses Kramdown, which probably has some subtle differences from GitHub Markdown.

> 
> How can I do a local build, is this documented on the site?

Yup, have a look at the bottom of https://hepsoftwarefoundation.org/jekyll-beginners.html. It’s usually pretty easy to get a local Jekyll running and then you can preview everything to check it. Otherwise, as you’ll have seen, it’s all too easy for mistakes to creep in; and as each fix is a commit, they leave a permanent trail… 

It also helps a lot if you’re asked to review someone else’s PR, as you can check their changes on your local machine.

> 
> By the way, is it ok if I self-approve these fixes?

Only if you checked them properly first 😄 

Cheers

Graeme